jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/interfaces/IsReadOnly.java

```diff
@@ -16,6 +16,6 @@ public interface IsReadOnly extends ICoreElement {
 
     @JDIAction("Check that '{name}' is readonly")
     default boolean isReadOnly() {
-        return core().attr("class").contains("--is-readonly");
+        return core().attr("class").contains("-readonly");
     }
```

Comment:
Hi :) Could you please help me with methods from task setValue/getValue? Failed to find them :(

Reply:
Hi:) We have methods **slideHorizontalTo(Double givenValue)** and **slideVerticalTo(String givenValue)**